test/busted/outputHandlers/nvim.lua

```diff
@@ -227,8 +227,19 @@ return function(options)
     return nil, true
   end
 
-  handler.testEnd = function(element, _parent, status, _debug)
+  local function write_status(element, string)
     local elapsedTime_ms = getElapsedTime(element)
+
+    -- XXX: not clear what that check is for.  Done in accc7a0b6.
+    -- Might be for "nan" - but `tonumber('nan') == tonumber('nan')`.
+    if elapsedTime_ms == elapsedTime_ms then
```

Comment:
Any idea about this?  (`elapsedTime_ms == elapsedTime_ms` - when would this be false?)
Otherwise I'll remove it.

Reply:
seems like a typo. Is there anything else that maybe ZyX meant to compare?

Reply:
Could not see anything - ref: https://github.com/neovim/neovim/commit/accc7a0b6cf1d2c036181be18a512b791f3ceb79#diff-d6cf5cb410d55194dd8651f7de32127dR254-R256